.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["es2015", "stage-0", "react"]
+  "presets": ["es2015", "stage-0", "react"],
```

Comment:
are we using `stage-0` still? might want to bump this up higher if we can get away with it